internal/delete_handler_test.go

```diff
@@ -0,0 +1,62 @@
+package internal
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/pkg/storages/memory"
+	"github.com/wal-g/wal-g/pkg/storages/storage"
+)
+
+func CreateMockStorageFolder() storage.Folder {
```

Comment:
Just moved corresponding tests along with the tested function `DeleteObjectsWhere`